src/ui/widgets/EmbeddedDisplay/bobParser.ts

```diff
@@ -245,10 +245,10 @@ export function parseBob(
       (pvName: ElementCompact): PV => opiParsePvName(pvName, defaultProtocol)
     ],
     font: ["font", bobParseFont],
-    rules: [
-      "rules",
-      (rules: Rule[]): Rule[] => opiParseRules(rules, defaultProtocol)
-    ],
+    // rules: [
```

Comment:
Can we remove this commented out code now?

Reply:
Ah yes, I meant to remove that. Thank you for spotting it